pubsubplus/templates/solaceConfigMap.yaml

```diff
@@ -157,17 +158,14 @@ data:
           "Local Active")
             echo "`date` INFO: ${APP}-Node activity status is Local Active, after ${run_time} seconds"
             # We should only be here on new cluster create, if not likely a bug
-            # Need to issue assert master to get back into sync"
-            resync_step="assert-master"
-            break
-            ;;
-          "Mate Active")
-            echo "`date` INFO: ${APP}-Node activity status is Mate Active, after ${run_time} seconds"
-            # This is normal state if we are backup or recreated later on
-            # will issue a resync master to get back into sync
-            resync_step="resync-master"
-            break
-            ;;
+            if [[ ! -e ${STARTUP_FILE} ]];
+            then
+                # Need to issue assert master to get back into sync only one time when the PubSub+ Event Broker starts the first time
+                echo "`date` INFO: ${APP}-Node attempting assert-master for firs time"
+                resync_step="assert-master"

```

Comment:
How about renaming the variable to "resync_step_required" and have it set to "true"